rust/src/delta.rs

```diff
@@ -771,6 +771,19 @@ impl DeltaTable {
         }
     }
 
+    /// Returns provenance information, including the operation, user, and so on, for each write to a table.
+    /// Table history is retained for 30 days.
```

Comment:
I believe this 30 is not hardcoded and largely depends on `delta.logRetentionDuration` config?

Reply:
Right, I will mention that the 30 days is by default!